ci/pipeline

```diff
@@ -221,3 +235,47 @@ def jenkins(): Unit = {
   if(utils.isPullRequest) pr()
   else master()
 }
+
+/**
+ * Executes the Community Release which includes:
+ *  1. tarball with version details
+ *  2. release of docker with that version
+ *  3. git tag of version (pushed to server)
+ * @param requestVersion The version attempting to be released v1.4.6
+ * @param gitSha The git commit sha.  This can be shorthand (ex. 0e1)
+ * @param latest Boolean to indicat if the Docker latest should be updated to this version.
+ * @return Version and artifact description of Marathon build.
+ */
+@main
+def release(requestVersion: String, gitSha: String, latest: Boolean = false): Unit = {
+
+  val releaseVersion = withoutVersionPrefix(requestVersion)
+  val tagVersion = s"v$releaseVersion"
```

Comment:
Hehe, ok, this reads funny now:

```scala
val releaseVersion = withoutVersionPrefix(requestVersion)
val tagVersion = s"v$releaseVersion"
```

one would wonder if `requestVersion == tagVersion`. I guess `withoutVersionPrefix` jus removes a prefix if there is one.